docs/parser.md

```diff
@@ -194,70 +195,69 @@ require("@babel/parser").parse("code", {
+| Name                                                                                                                                      | Code Example                                                     |
+| ----------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `asyncDoExpressions` ([proposal](https://github.com/tc39/proposal-async-do-expressions))                                                  | `async do { await requestAPI().json() }`                         |
+| `decimal` ([proposal](https://github.com/tc39/proposal-decimal))                                                                          | `0.3m`                                                           |
+| `decorators` ([proposal](https://github.com/tc39/proposal-decorators)) <br/> `decorators-legacy`                                          | `@a class A {}`                                                  |
+| `decoratorAutoAccessors` ([proposal](https://github.com/tc39/proposal-decorators))                                                        | `class Example { @reactive accessor myBool = false; }`           |
+| `deferredImportEvaluation` ([proposal](https://github.com/tc39/proposal-defer-import-eval))                                               | `import defer * as ns from "dep";`                               |
+| `destructuringPrivate` ([proposal](https://github.com/tc39/proposal-destructuring-private))                                               | `class Example { #x = 1; method() { const { #x: x } = this; } }` |
+| `doExpressions` ([proposal](https://github.com/tc39/proposal-do-expressions))                                                             | `var a = do { if (true) { 'hi'; } };`                            |
+| `explicitResourceManagement` ([proposal](https://github.com/tc39/proposal-explicit-resource-management))                                  | `using reader = getReader()`                                     |
+| `exportDefaultFrom` ([proposal](https://github.com/tc39/ecmascript-export-default-from))                                                  | `export v from "mod"`                                            |
+| `functionBind` ([proposal](https://github.com/zenparsing/es-function-bind))                                                               | `a::b`, `::console.log`                                          |
+| `functionSent` ([proposal](https://github.com/tc39/proposal-function.sent))                                                               | `function.sent`                                                  |
+| `importAttributes` ([proposal](https://github.com/tc39/proposal-import-attributes)) [<br/> `importAssertions` (⚠️ deprecated)](:::babel7) | `import json from "./foo.json" with { type: "json" };`           |
```

Comment:
I think it would be very confusing hacking around the link syntax for conditional rendering. How about we set up a new table listing the deprecated parser plugins and toggle off that table in Babel 8?